src/elli/importer/accurion.py

```diff
@@ -0,0 +1,99 @@
+"""A helper class to load data from Accurion EP4 DAT files.
+Typical files look like: Si3N4_on_4inBF33_W03_20240905-105631.ds.dat
+"""
+
+import pandas as pd
+import chardet
+import numpy as np
+
+def detect_encoding(fname: str):
+	r"""Detects the encoding of file fname. Used in read_accurion_psi_delta only.
+  Args:
+    fname (str): Filename
+  """
+	with open(fname, 'rb') as f:
+		raw_data = f.read()
+	result = chardet.detect(raw_data)
+	return result['encoding']
+
+def read_accurion_psi_delta(fname: str, wrap=True) -> pd.DataFrame:
+	r"""Read a psi/delta Accurion dat file.
+	
+  Args:
+    fname (str): Filename of the measured dat file
+    wrap (bool): Whether delta values should be wrapped; should always be True
+
+  Returns:
+    pd.DataFrame: DataFrame containing the psi/delta data in the pyElli-compatible format.
+  """
+	encoding = detect_encoding(fname)
+	psi_delta_df = pd.read_csv(fname, delimiter="\t", encoding=encoding, skiprows=0, header=0)[1:].astype(float)
+	psi_delta_df = psi_delta_df.reindex(columns=list(["AOI", "Lambda", "Delta", "Psi"]))
+	psi_delta_df = psi_delta_df.rename(columns={
+		"AOI": "Angle of Incidence",
+		"Lambda": "Wavelength",
+		"Delta": "Δ",
+		"Psi": "Ψ"
+		})
+	psi_delta_df = psi_delta_df.groupby(["Angle of Incidence", "Wavelength"]).sum()
+
+	# wrap delta range
+	if wrap:
+		psi_delta_df.loc[:, "Δ"] = psi_delta_df.loc[:, "Δ"].where(
+			psi_delta_df.loc[:, "Δ"] <= 180, psi_delta_df.loc[:, "Δ"] - 360
+		)
+
+	return psi_delta_df
+
+def convert_psi_delta_to_isotropic_mueller_matrix(psi_delta_df) -> pd.DataFrame:
```

Comment:
I think it makes sense to move this function to `src/elli/utils.py` as this is a general function which is not particular for this reader.

Reply:
I think it would also make sense to add some tests for this. But I can also add them if you like.